docs/markdown/snowplow_media_player_common_cols.md

```diff
@@ -98,16 +98,82 @@ The `derived_tstamp` denoting the time when the event started.
 The `derived_tstamp` denoting the time when the last media player event belonging to the specific level of aggregation (e.g.: page_view by media) started.
 {% enddocs %}
 
-{% docs col_play_time_sec %}
-Estimated duration of play in seconds. It is calculated using the percent_progress events that are fired during play. In case such an event is fired, it is assumed that the total section of the media in between the previous and current percent_progress is played through, even if the user seeks to another point in time within the audio / video. The more often these events are tracked (e.g. every 5% of the media's length) the more accurate the calculation becomes.
+{% docs col_play_time_secs %}
+Total seconds user spent playing content (excluding linear ads). If the media session entity is tracked with media events, the information is read from there. This is an accurate measurement provided by the tracker.
```

Comment:
I wonder if it is worth to group the total time related fields according to whether the time spent on them is calculated counting only one watches per user or rewatches as well. I think it was already problematic in the initial version but now users might wonder which tracking solution to implement for more accurate metrics depending on the calculations behind the fields they can get using them. Although I think it would belong to our internal docs it would be nice to see it come together during the PR in case some of the logic should be changed. (E.g. changing field names to reflect this somehow or if any field is missing, eg. total time including rewatches in media stats (although this might be there already)

Reply:
So we now have two columns:

1. `play_time_secs` – which is the total play time including rewatching the same content twice
2. `content_watched_secs` and `content_watched_percent` – the reflect the unique duration of content that was watched ignoring rewatched content.

These are reported both in the media base and media stats tables. Have added a note in the docs to highlight that the play time counts rewatched content while the content watched time does not. Does that make it clear enough or do you have something else in mind?

Reply:
Thanks for explaining and adding those bits in! I think this is a good selling point why to upgrade and use v2 (for release notes I mean). I'm inclined to propose to also add the v2 dependency in the column docs as those definitions are not package dependent and they might be confused why they get nulls for that.

Reply:
Do you mean to not talk about the option to read the play time from the media session in case they don't have the media session context enabled? I'm a bit worried about that, because then they will never know that there is a possibility to have better data. I think we can do a bit of marketing in the docs to let them know that they can get more accurate measurement if they upgrade to the v2 media schemas.

Reply:
No no, the opposite, advertise it in the release! And I mean to add to the markdown docs that `content_watched_secs` and `content_watched_secs` that would be null if they used v1 so that they are warned not to expect them and promote upgrade. (also for us to debug, remember later etc)

Reply:
Ahh, I see! Have added the note that these properties will be null if using older schemas.

Reply:
Excellent, thank you! From this commit I can clearly see the affected fields in one place, which helps!